example-with-touch.yaml

```diff
@@ -1,5 +1,11 @@
 esphome:
   name: s3
+  platformio_options:
+    upload_speed: 2000000
+    board_build.f_flash: 80000000L
+    board_upload.flash_size: 16MB
+    board_build.partitions: default_16MB.csv
+    board_build.arduino.memory_type: qio_opi
```

Comment:
Is this really needed? Especially the upload speed shouldn't affect most users using OTA updates.

Reply:
I removed the upload speed. The other ones are necessary to use the board fully.

Reply:
If you don't set the flash size and partition you only use 4MB of the 16MB of the T-Display-S3.
If you don't set flash speed and memory type the flash and especially the PSRam performance is lower.